homeassistant/components/light/tradfri.py

```diff
@@ -80,21 +82,35 @@ def brightness(self):
         """Brightness of the light (an integer in the range 1-255)."""
         return self._light_data.dimmer
 
+    @property
+    def rgb_color(self):
+        """RGB color of the light."""
+        return self._rgb_color
+
     def turn_off(self, **kwargs):
         """Instruct the light to turn off."""
         return self._light_control.set_state(False)
 
     def turn_on(self, **kwargs):
-        """Instruct the light to turn on."""
+        """
+        Instruct the light to turn on.
+
+        After adding "self._light_data.hexcolor is not None"
+        for ATTR_RGB_COLOR, this also supports Philips Hue bulbs.
+        """
         if ATTR_BRIGHTNESS in kwargs:
             self._light.light_control.set_dimmer(kwargs.get(ATTR_BRIGHTNESS))
+        if ATTR_RGB_COLOR in kwargs and self._light_data.hex_color is not None:
+            hex_color = '{0:02x}{1:02x}{2:02x}'.format(*kwargs[ATTR_RGB_COLOR])
```

Comment:
Can you extract and move this to the `util/color.py` file and include a quick test for it